dir.props

```diff
@@ -79,9 +79,10 @@
   <Import Project="$(ToolsDir)Build.Common.props" Condition="Exists('$(ToolsDir)Build.Common.props')" />
 
   <PropertyGroup>
-     <ArmEnvironmentVariables Condition="'$(ArmEnvironmentVariables)' == ''">ROOTFS_DIR=$(BaseIntermediatePath)crossrootfs/arm</ArmEnvironmentVariables>
-     <ArmEnvironmentVariables Condition="'$(Platform)' == 'armel'">ROOTFS_DIR=$(BaseIntermediatePath)crossrootfs/armel</ArmEnvironmentVariables>
-   </PropertyGroup>
+     <RootFsEnvironmentVariables Condition="'$(RootFsEnvironmentVariables)' == ''">ROOTFS_DIR=$(BaseIntermediatePath)crossrootfs/arm</RootFsEnvironmentVariables>
+     <RootFsEnvironmentVariables Condition="'$(Platform)' == 'armel'">ROOTFS_DIR=$(BaseIntermediatePath)crossrootfs/armel</RootFsEnvironmentVariables>
+     <RootFsEnvironmentVariables Condition="'$(Platform)' == 'x86'">ROOTFS_DIR=$(BaseIntermediatePath)crossrootfs/x86</RootFsEnvironmentVariables> 
```

Comment:
With the other change to `<BuildCommand Condition="'$(RootFsEnvironmentVariables)' != ''">$(RootFsEnvironmentVariables)`, I think you'll have to change these to make sure `RootFsEnvironmentVariables` is only defined on platforms that need rootfs. This is what comes to mind but I'm not sure:

```xml
  <PropertyGroup Condition="'$(RootFsEnvironmentVariables)' == ''">
     <RootFsEnvironmentVariables Condition="$(Platform.Contains('arm'))">ROOTFS_DIR=$(BaseIntermediatePath)crossrootfs/arm</RootFsEnvironmentVariables>
     <RootFsEnvironmentVariables Condition="'$(Platform)' == 'armel'">ROOTFS_DIR=$(BaseIntermediatePath)crossrootfs/armel</RootFsEnvironmentVariables>
     <RootFsEnvironmentVariables Condition="'$(OS)' != 'Windows_NT' AND '$(Platform)' == 'x86'">ROOTFS_DIR=$(BaseIntermediatePath)crossrootfs/x86</RootFsEnvironmentVariables> 
  </PropertyGroup>
```

Reply:
LGTM